.chloggen/elasticexporter-fix-custom-header.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fixed nil panic error when setting custom headers in elasticsearch exporter
+
+# One or more tracking issues related to the change
+issues: []
```

You need to put an issue/PR here:
```suggestion
issues: [16017]
```